src/presentation/http/router/note.ts

```diff
@@ -55,9 +60,8 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   /**
    * Get note by id
    */
-  fastify.get<{ Params: GetNoteByIdOptions }>('/:id', async (request, reply) => {
+  fastify.get<{ Params: GetNoteByIdOptions }>('/:id', { preHandler: opts.authMiddleware }, async (request, reply) => {
```

Comment:
maybe this route should not require authorisation?

Reply:
removed